modules/index/src/test/java/org/apache/ignite/internal/index/IndexBuildControllerTest.java

```diff
@@ -112,13 +117,26 @@ void setUp() {
         catalogManager = createCatalogManagerWithTestUpdateLog(NODE_NAME, clock);
         assertThat(catalogManager.startAsync(new ComponentContext()), willCompleteSuccessfully());
 
+        PartitionReplicaLifecycleManager partitionReplicaLifecycleManager = mock(PartitionReplicaLifecycleManager.class);
+        ZonePartitionResources zonePartitionResources = mock(ZonePartitionResources.class);
+        doReturn(zonePartitionResources).when(partitionReplicaLifecycleManager).zonePartitionResourcesOrNull(any());
+
+        ZonePartitionReplicaListener replicaListener = mock(ZonePartitionReplicaListener.class);
+        doReturn(completedFuture(replicaListener)).when(zonePartitionResources).replicaListenerFuture();
+
+        TableTxRwOperationTracker txRwOperationTracker = mock(TableTxRwOperationTracker.class);
+        //noinspection unchecked
+        PendingComparableValuesTracker<HybridTimestamp, Void> safeTime = mock(PendingComparableValuesTracker.class);
```

Comment:
I would suggest to inject the mocks using the `@Mock` annotation in order to avoid the "unchecked" problem, but that's not a big deal